modules/deploying-a-pod-that-includes-an-aws-sdk.adoc

```diff
@@ -45,6 +49,10 @@ spec:
     - "sleep 100000" <5>
   terminationGracePeriodSeconds: 0
   restartPolicy: Never
+  securityContext:
```

Comment:
@bergerhoffer the spacing doesn't looks correct here, Could you please take a look?

Reply:
Moved up